pkg/server/server.go

```diff
@@ -110,6 +112,10 @@ func (s *Server) ListObjects(ctx context.Context, req *openfgapb.ListObjectsRequ
 		return nil, err
 	}
 
+	if commands.IsAuthorizationModelObsolete(model.SchemaVersion, s.config.AllowEvaluatingObsoleteModels) {
+		return nil, serverErrors.ObsoleteAuthorizationModel
+	}
+
```

Comment:
In all the other commands, this validation occurs inside the commands themselves. Why is this one for ListObjects at the server level? Can we agree on either doing this inside the commands or outside?

Reply:
The check itself does not have anything inside the commands.  But for unity sake, I can move it outside

Reply:
Unfortunately, we only know the details of the model inside each command.  Do we want to move all the reading of auth model from the server level to the server level?  If we are to do this refactoring, we probably want to make the check/list objects/other command matches.  (And that itself may be done in a separate work)